### ISSUE TITLE
D5 :: free-form css are wrapped in {} on frontend

### ISSUE THREAD

**Issue by elzadj** [elegantthemes/Divi-Beta#100]
### Problem Description
1. Describe what you did
Any free-form css we add are wrapped in {} on frontend in ```et-critical-inline-css```, i.e. they don't work. Happens if we use 'selector', or a class for that module. Backend is fine. 

2. Describe what you expected to happen
Frontend css should look somewhat like this:

```
<style id="et-critical-inline-css">
...
.et-db #et-boc .et-l.et-l--post .et_builder_inner_content{z-index:auto}
.test-person-section-class .et_pb_team_member{width:200px}
</style>
```
3. Describe what actually happened
But we get this: 

```
<style id="et-critical-inline-css">
...
.et-db #et-boc .et-l.et-l--post .et_builder_inner_content{z-index:auto}
{ 
     .test-person-section-class .et_pb_team_member{width:200px}
}
</style>
```

### Detailed Steps To Reproduce
Please provide a detailed list of exact steps that reproduce the issue.
1. Add any module
2. Add free-form css
3. Check frontend

### Screenshot and/or gif
Here is a video with a blurb:

[d5-person-test-MaxiBlocks.webm](https://github.com/user-attachments/assets/edee45e5-3b0e-4e74-ab1d-13030e6d419a)

### System Information
* Divi version: 5.0.0-public-alpha.1
* Operating System: Ubuntu 24.04.2
* Browser: Chrome 129.0.6668.100 (Official Build) (64-bit)
* PHP version: 8.3.6
* MySQL version: 8.0.39-0ubuntu0.24.04.2 for Linux on x86_64 ((Ubuntu))
* Web Server: Apache/2.4.58 (Ubuntu)
* WordPress version: 6.6.2

**Comment by elzadj** [comment]
Note: not fixed in 5.0.0-public-alpha.1.1

**Comment by elzadj** [comment]
Fixed in public alpha 2 or 3
